static/app/components/surfaceProvider.tsx

```diff
@@ -0,0 +1,25 @@
+import {createContext, type ReactNode, useContext} from 'react';
+
+const SurfaceContext = createContext<string | undefined>(undefined);
+
+export function useSurface(): string {
+  const surface = useContext(SurfaceContext);
+  if (!surface) {
+    // eslint-disable-next-line no-console
+    console.error('Undefined surface! Please use SurfaceProvider as a context provider.');
+  }
+  return surface || '';
+}
```

Comment:
i would drop all this, we could either send it to sentry, or not log anything at all because idk who will read this in the console otherwise. 

```suggestion
  return useContext(SurfaceContext);
}
```

Reply:
ya, just drop this whole hook, callsites can type `useContext(SurfaceContext);` each time